engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStore.scala

```diff
@@ -119,7 +126,8 @@ final case class ExecutionStore(private val statusStore: Map[JobKey, ExecutionSt
 
   private def arePrerequisitesDone(key: JobKey): Boolean = {
     lazy val upstreamAreDone = key.scope.upstream forall {
-      case n @ (_: WdlCall | _: Scatter | _: Declaration) => upstreamIsDone(key, n)
+      // TODO WOM: Declaration ??
```

Comment:
Now expressed as an `ExpressionNode`.
We also have `ConditionalNode` which should probably be here too?

Reply:
Yep, I'm going to leave this for a next PR though. I made tickets to restore declarations and conditionals